README.md

```diff
@@ -85,7 +85,7 @@ invoke deps
 
 #### Updating `poetry.lock` dependencies
 
-The dependencies installed in our CI and the docker build step are determined by the [poetry.lock file](https://python-poetry.org/docs/basic-usage/#installing-with-poetrylock).
+The dependencies installed in our CI and the Docker build step are determined by the [poetry.lock file](https://python-poetry.org/docs/basic-usage/#installing-with-poetrylock).
```

Comment:
Just a few capitalization updates for consistency here; main change is at the bottom of this file